server.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import FastAPI
+from routes import basic_db_functions_route
+
+app = FastAPI()
+
```

Comment:
remove exssesive line spaces!

Reply:
I do it intentionally for having a better look for the code, it’s more readable as from my point of view when I leave line spaces. I deal with code as chunks, when I finish a specific chunk I intentionally leave a line space to announce a start of a new chunk.

Reply:
That’s also what I see all youtubers and coders do, and also any code that ChatGPT provides, which means it’s a practice among coders.